samples/net/cloud/tagoio_http_post/sample.yaml

```diff
@@ -15,5 +15,5 @@ tests:
     extra_args: SHIELD=esp_8266_arduino OVERLAY_CONFIG="overlay-wifi.conf"
     platform_allow: frdm_k64f nucleo_f767zi
   sample.net.cloud.tagoio_http_post.modem:
-    extra_args: OVERLAY_CONFIG="overlay-modem.conf"
+    extra_args: OVERLAY_CONFIG="overlay-modem.conf" DTC_OVERLAY_FILE="boards/uart0_dts.overlay"
```

Comment:
It is necessary differentiate board definition for shields. This means that it is necessary two entries:

```
  sample.net.cloud.tagoio_http_post.modem:
    extra_args: OVERLAY_CONFIG="overlay-modem.conf"
    platform_allow: sam4e_xpro
  sample.net.cloud.tagoio_http_post.modem.arduino:
    extra_args: OVERLAY_CONFIG="overlay-modem.conf" DTC_OVERLAY_FILE="arduino.overlay"
    platform_allow: frdm_k64f
```
Create a file at **\<zephyr\>/samples/net/tagoio_http_post/arduino.overlay** with following content:
```
&arduino_serial {
	status = "okay";
	current-speed = <115200>;

	gsm: gsm-modem {
		compatible = "zephyr,gsm-ppp";
		label = "gsm_ppp";
	};
};

```